contrib/terraform/common/common.tf

```diff
@@ -0,0 +1,93 @@
+variable "temp_basedir" {
+  type    = "string"
+  default = "/tmp"
+}
+
+variable "inventory_deploy" {
```

Comment:
Would it make sense to add a little comment (if possible) documenting what these variables are used for?

Reply:
Indeed, I'll do a big pass on documentation